lib/LDKeyPair.js

```diff
@@ -138,6 +115,52 @@ class LDKeyPair {
   verifyFingerprint(/* {fingerprint} */) {
     throw new Error('Abstract method, must be implemented in subclass.');
   }
+
+  /* eslint-disable max-len */
+  /**
+   * Returns a signer object for use with
+   * [jsonld-signatures]{@link https://github.com/digitalbazaar/jsonld-signatures}.
+   * NOTE: Applies only to verifier type keys (like ed25519).
+   *
+   * @example
+   * > const signer = keyPair.signer();
+   * > signer
+   * { sign: [AsyncFunction: sign] }
+   * > signer.sign({data});
+   *
+   * @returns {{sign: Function}} A signer for json-ld usage.
+   */
+  /* eslint-enable */
+  signer() {
+    return {
+      async sign({/* data */}) {
```

Comment:
I can't recall -- but does the signer API also need an `id` property? I feel as though there's an `id` property used to populate things like `verificationMethod` when using a signer.

Reply:
@dlongley - you're thinking of the higher-level jsigs.sign() functionality (that uses the key id for verificationMethod). This `signer()` is much more lower level, and uses the built-in node crypto to sign a buffer, so it doesn't use an id.

Reply:
https://github.com/digitalbazaar/ed25519-verification-key-2020/blob/main/lib/Ed25519VerificationKey2020.js#L212